api/test/shell/cli_test.sh

```diff
@@ -93,3 +93,57 @@ if [[ `grep -c "INFO" ./error.log` -eq '0' ]]; then
     echo "failed: failed to write log on right level"
     exit 1
 fi
+
+
+# etcd basic auth
+# add root user
+curl -L http://localhost:2379/v3/auth/user/add \
+  -X POST -d '{"name": "root", "password": "root"}'
```

Comment:
Actually when you use `-d` option, the method is set to `POST` by default.

Reply:
ok